source/features/clean-conversation-sidebar.tsx

```diff
@@ -40,9 +40,14 @@ function cleanSection(selector: string): boolean {
 
 	const heading = select(':scope > details, :scope > .discussion-sidebar-heading', container)!;
 
-	// Magic. Do not touch.
-	// Section is empty if: no sibling element OR empty sibling element
-	if (heading.nextElementSibling?.firstElementChild) {
+	const labelsUniqueSelector = '.IssueLabel';
+	const mileStonesUniqueSelector = '.Progress-item';
+	const developmentsUniqueSelector = '[data-hovercard-type]';
+	const projectsUniqueSelector = '.octicon';
```

Comment:
It icon is too generic, it could even be the close button. I’d also review line 45